pkg/volume/util/atomic_writer.go

```diff
@@ -103,8 +103,10 @@ const (
 //  6. The payload is written to the new timestamped directory
 //     7.  A symlink to the new timestamped directory ..data_tmp is created that will
 //     become the new data directory
-//     8.  The new data directory symlink is renamed to the data directory; rename is atomic
-//     9.  Symlinks and directory for new user-visible files are created (if needed).
+//
+//  8. Set the permissions of the files.
```

Comment:
This whole comment block is weird, but if we are sticking to style, I'd prefer this as:

```
6.  The payload is written to the new timestamped directory
    7. Permissions are set (if setPerms is not nil) on the new timestamped directory and files
    8. A symlink to ...
    9. The new data directory symlink ...
    10. Symlinks and directory for new ...
```

In particular, I think setPerms should be after write, BEFORE symlink

Reply:
Fixed. 

The was some formatting that confused `gofmt` and it did not parse the numbered list anymore as numbered list properly.

I changed the order of setPerms and creation of `..data_tmp` symlink.